mers/clearlinux/README.md

```diff
@@ -0,0 +1,45 @@
+# Media Reference Stack
+
+## Building Locally
+
+The Dockerfiles for all Intel System Stacks container images are available at [stacks repository](https://github.com/intel/stacks). These can be used to build and modify the container images.
```

Comment:
Can we add a paragraph that explicitly says the following command is for building the image?

Reply:
yes